packages/api/src/main/messaging/config.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/**
+ * Each service gets a queue. Services bind their queue to one or more exchanges to receive
+ * messages.
+ */
+enum Queue {
+  Github = 'q_github_v1',
+  Logging = 'q_logging_v1',
+  Search = 'q_search_v1'
+}
```

Comment:
are we using this anywhere?

Reply:
Not in this PR, but the logging one is used over [here](https://github.com/carbon-design-system/carbon-platform/blob/9ca0958b008ed2f72d573834971d6d87474495bb/services/logging/src/main/index.ts)